package.json

```diff
@@ -31,12 +31,13 @@
   },
   "dependencies": {
     "@activeviam/activeui-sdk-5.0": "npm:@activeviam/activeui-sdk@5.0.15",
-    "@activeviam/activeui-sdk-5.1": "npm:@activeviam/activeui-sdk@5.1.0-20221220092919-bb816dc",
+    "@activeviam/activeui-sdk-5.1": "npm:@activeviam/activeui-sdk@5.1.0-20221220105623-9a2eef6",
```

Comment:
I just realized that I had actually taken the parent commit and not the ocmmit itself 🤦 
At least this allowed to isolate the changes necessary to make Jest work with ESM packages.

Reply:
Shouldn't it be [this commit](https://github.com/activeviam/activeui/commit/15897ed90c038f5a452548b5561ac6dae38af4c8) though: 15897ed90c038f5a452548b5561ac6dae38af4c8?